packager_test.go

```diff
@@ -75,7 +75,10 @@ func TestSameDomain(t *testing.T) {
 	defer server.Close()
 
 	pkg := webpackager.NewPackager(makeConfig(server))
-	pkg.Run(urlutil.MustParse("https://example.org/hello.html"), date)
+	err := pkg.Run(urlutil.MustParse("https://example.org/hello.html"), date)
+	if err != nil {
+		t.Fatal(err)
```

Comment:
This is an error from the test subject (system-under-test), so this t.Fatal should have a message:

```go
t.Fatal("pkg.Run() = error(%q), want success", err)
```

Reply:
Done 5x with `Fatalf`.